README.md

```diff
@@ -1,46 +1,95 @@
-The PHP Interpreter
-===================
+<div align="center">
+    <a href="https://php.net">
+        <img
+            alt="PHP"
+            src="https://static.php.net/www.php.net/images/logos/new-php-logo.svg"
+            width="150">
+    </a>
+</div>
 
-This is the github mirror of the official PHP repository located at
-https://git.php.net.
+# The PHP Interpreter
 
-[![Build Status](https://secure.travis-ci.org/php/php-src.svg?branch=master)](http://travis-ci.org/php/php-src)
+PHP is a popular general-purpose scripting language that is especially suited to
+web development. Fast, flexible and pragmatic, PHP powers everything from your
+blog to the most popular websites in the world. PHP is distributed under the PHP
+License v3.01.
+
+[![Build status](https://travis-ci.org/php/php-src.svg?branch=master)](https://travis-ci.org/php/php-src)
 [![Build status](https://ci.appveyor.com/api/projects/status/meyur6fviaxgdwdy?svg=true)](https://ci.appveyor.com/project/php/php-src)
 
-Pull Requests
-=============
-PHP accepts pull requests via github. Discussions are done on github, but
-depending on the topic can also be relayed to the official PHP developer
-mailing list internals@lists.php.net.
+## Documentation
+
+The PHP manual is available online at https://php.net/docs.
+
+## Installation
+
+### Prebuilt packages and binaries
+
+Prebuilt packages and binaries can be used to get up and running fast with PHP.
+
+For Windows, the PHP binaries can be obtained from https://windows.php.net.
+After extracting the archive the `*.exe` files are ready to use.
+
+For other systems, see the documentation at https://php.net/install.
+
+### Building PHP from source
+
+On \*nix systems:
 
-New features require an RFC and must be accepted by the developers.
-See https://wiki.php.net/rfc and https://wiki.php.net/rfc/voting for more
+    ./buildconf
+    ./configure
+    make
```

Comment:
There's a `-j` param that we can pass to this command to make it parallel and fast. In Linux is the result of `nproc` and in macOS `sysctl -n hw.physicalcpu on Mac OS`, both in terminal.

Maybe we can add a comment after the Windows link:

> Note: if you want to make this installation fast, you can pass a `-j` parameter to `make`, which is your number of cores + 1.

Reply:
Configure and make options are definitely the first thing that someone who will install this, will want to check... I've added another quick intro paragraph on that. For the configure and make options there will need to be another document (documentation chapter somewhere updated), that's for sure... It's the next step for sure. Thanks for the hints.